pkgs/servers/miniflux/default.nix

```diff
@@ -11,7 +11,7 @@ in buildGoModule {
     owner = pname;
     repo = "v2";
     rev = version;
-    sha256 = "sha256-a27eKOhW2vHmPktLgqHKqiwtC9T6GRwnOeNReeMsaeM=";
+    sha256 = "sha256-SJw9C6s2JaI2AOVDazn1+N23Yp+Ul5P239tdAzXOxv4=";
   };
 
   vendorHash = "sha256-Oe7el4tE/gwI6qL/fjJgnv1jbNSKrCnq1nBq+dD7Gik=";
```

Comment:
```suggestion
  vendorHash = "sha256-t1wFxXws3411/ZVNF6D4g7hDCSbmpCJhh8JCJ4QJdn4=";
```

Reply:
Strange, I used `nix-update` but somehow got the hash wrong.